build.gradle

```
@@ -143,6 +145,10 @@ subprojects { subproj ->
         maven {
             url 'https://jitpack.io'
         }
+        //PEconomy 
+        maven {
+             url "https://raw.github.com/SoKnight/PEconomy/mvn-repo/"
+        }
```

```suggestion

        //PEconomy 
        maven {
             url "https://raw.github.com/SoKnight/PEconomy/mvn-repo/"
        }
```
Add a newline above to keep consistent formatting